ibis/omniscidb/client.py

```diff
@@ -696,6 +696,27 @@ def drop_view(self, name, database=None):
         statement = ddl.DropView(name, database=database)
         self._execute(statement, False)
 
+    def create_table_from_csv(self, path, table_name, schema, database=None):
+        """
+        Wraps the CRREATE TABLE WITH  DML statement.
+        Loads data into an Omniscidb table from CSV files
+
+        Parameters
+        ----------
+        path : string
+          Path to the input CSV file
+        table_name : string
+          Table name to create and load data
+        schema : ibis.Schema
+          Column description
+        database : string, optional
+        Returns
+        -------
+        query : OmniSciDBQuery
+        """
+        stmt = ddl.CreateTableFromCSV(path, table_name, schema, database)
+        return self._execute(stmt)
+
     def create_table(
         self, table_name, obj=None, schema=None, database=None, max_rows=None
```

Comment:
Why not just implement the pandas.DataFrame type within this create_table function?

Reply:
@scottcode does this comment make sense to you? https://github.com/ibis-project/ibis/pull/2014#issuecomment-547185909

Reply:
I think I misunderstood, as I'm not too familiar with Omnisci. Is this about loading the contents of a CSV accessible to the Python process and uploading it as a table in the database? If so, then it would make more sense to have the user read in the csv using `pandas.read_csv()` and then implement the `pandas.DataFrame` version of the `create_table` method. 

Or is this about creating an external table definition or specifying table storage as a CSV? If so, then maybe we could just put an extra keyword argument in the existing `create_table` and `CreateTableWithSchema` signatures.

Reply:
This is about creating an external "in memory" table from specific storage (e.g CSV).  Putting an extra keyword argument in the existing create_table and CreateTableWithSchema signatures makes sense.